src/App.tsx

```diff
@@ -1,8 +1,9 @@
-import React from 'react';
+import { Component } from 'react';
+import classNames from 'classnames';
 import 'bulma/css/bulma.css';
 import './App.scss';
 
-export const goodsFromServer = [
+export const goodsFromServer: string[] = [
```

Comment:
it's extra, ts already know about this

```suggestion
export const goodsFromServer = [
```